nomad/structs/config/vault.go

```diff
@@ -83,6 +84,46 @@ type VaultConfig struct {
 
 	// TLSServerName, if set, is used to set the SNI host when connecting via TLS.
 	TLSServerName string `mapstructure:"tls_server_name"`
+
+	// Servers-only fields.
+
+	// UseIdentity defines if workload identities should be used to derive
+	// Vault tokens.
+	//
+	// It is a transitional field used only during the adoption period of
+	// workload identities and will be ignored and removed in future versions.
+	UseIdentity *bool `mapstructure:"use_identity"`
```

Comment:
Arguably this new field could immediately be placed in the "Deprecated" section below.

Reply:
Hum...good point. I think I think I will leave it it here for now though as it still kind of expected to be set until 1.9 at least.